handbook/company/leadership.md

```diff
@@ -410,7 +410,36 @@ When it comes to performance feedback, [speak freely](https://fleetdm.com/handbo
-3. When you meet with your manager for your 1:1, periodically provide an update on how each of your direct reports is doing at the top of your own "Performance management" section in your 1:1 agenda doc.  
+3. When you meet with your manager for your 1:1, periodically provide an update on how each of your direct reports is doing at the top of your own "Performance management" section in your 1:1 agenda doc.
+
+## Skip Level 1:1 Meetings 
+
+At this time of the organization, most of the Skip Level 1:1s will be defined as non-managerial Fleeties meeting with the CEO. As the org grows, additional levels of management between team members and the CEO may be created, this section will be updated as needed with the appropriate links and language to reflect such a change.
+
+### The Why
+
+1.  Team members **must** feel absolutely heard by the CEO for them to resonate with the company’s core values.
+2.  A CEO needs to have **confidence** that his manager’s direct reports feel comfortable and supported when addressing sensitive topics(i.e. management) that could very well have dire repercussions if not addressed. A company that facilitates a culture where direct reports are fearful of telling the truth can only fail slowly and miserably.
+3. Non-managerial team members have low-level insight into their duties and responsibilities that may help Illuminate strategies that better assist themselves, their team, or other departments.
+4.  The CEO may discover insightful details regarding managers from a different perspective that may help them better support these managers.
+
+### The How-To
+
+**Actions For The Manager:**
+
+1.  Once communication from a team member has been received that they wish to initiate a Skip-Level 1:1 OR you would like to initiate a Skip-Level 1:1 on behalf of a Fleetie, you may do so by sending `@mikermcneil` a DM in Fleet Slack, at-mention our CEO in the [#help-leadership channel](https://fleetdm.slack.com/archives/C02HWSTJ17Z), or [schedule time with the CEO](https://fleetdm.com/handbook/company/communications#schedule-time-with-the-ceo).
+2.  In your communication to your direct report, be sure to link them to this section of the document so they have a clearer understanding of the Skip-Level 1:1 process, meeting structure, and helpful recommendations. 
+3.  After initiation is made with the CEO, managers should then make sure to help the CEO, and Fleetie establish communication to ensure a timeline has been set in motion for the two to meet. Managers then only need to check in with their direct reports to make sure everything is going accordingly.
+
+**Recommendations For The Non-Manager Fleetie:**
+
+1.  Take a breath first and read through the [CEO’s Flaws](#ceo-flaws). This will hopefully give you a better understanding of Mike and should hope to serve both you and Mike in making the conversation a smoother experience.
```

Comment:
- Always use the conical version of links (e.g. https://www.fleetdm.com/handbook/company/leadership#ceo-flaws), this prevents breaking links if this content ever moves to another page (which happens more than you would think).

```suggestion
```